lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -118,6 +131,7 @@ sub press_accept_button {
 sub open_clone_partition_dialog {
     my ($self) = @_;
     assert_screen(EXPERT_PARTITIONER_PAGE);
```

Comment:
Here, I would check that Overview Tab is selected (i.e. `assert_screen(OVERVIEW_TAB)`) and remove the last `assert_screen(OVERVIEW_TAB);` from `select_overview_tab` method to have consistency: first ensure the required screen is shown, then do action.

Reply:
Makes sense.